tests/openvino/test_quantization.py

```diff
@@ -119,16 +119,26 @@ class OVQuantizerTest(unittest.TestCase):
         (
             OVModelForCausalLM,
             "llama",
-            OVQuantizationConfig(
+            dict(
                 dataset="wikitext2",
                 num_samples=1,
-                weight_only=False,
-                weight_format="f8e4m3",
                 activation_format="f8e4m3",
             ),
             (13,),
             (16,),
         ),
+        (
+            OVModelForCausalLM,
+            "llama",
+            dict(
+                weight_quantization_config=dict(bits=4, weight_format="nf4", group_size=16),
+                full_quantization_config=dict(activation_format="f8e4m3"),
```

Comment:
What is the weight format in this case when you define just `activation_format`?

Reply:
Discussed offline: weights will also be quantized to `activation_format` data type